cluster/juju/layers/kubeapi-load-balancer/layer.yaml

```diff
@@ -1,9 +1,12 @@
+repo: https://github.com/kubernetes/kubernetes.git
 includes:
-  - layer:nginx
-  - layer:tls-client
-  - interface:public-address
-repo: https://github.com/juju-solutions/kubernetes-load-balancer
+  - 'layer:nginx'
+  - 'layer:tls-client'
+  - 'interface:public-address'
 options:
+  basic:
+    packages:
+      - 'nfs-common'
```

Comment:
OK I can remove it. I was worried this would come in a request in the future if we did not have it.